How-To-Setup-Internal-Package-Repository.md

```diff
@@ -271,34 +343,39 @@ Jenkins requires several PowerShell scripts to automate the processes. Create a
   . .\ConvertTo-ChocoObject.ps1
 
   # get all of the packages from the test repo
-  $testPkgs = choco list --source $TestRepo | Select-Object -Skip 1 | Select-Object -SkipLast 1 | ConvertTo-ChocoObject
-  $prodPkgs = choco list --source $ProdRepo | Select-Object -Skip 1 | Select-Object -SkipLast 1 | ConvertTo-ChocoObject
+  $testPkgs = choco.exe list --source $TestRepo -r | ConvertTo-ChocoObject
+  $prodPkgs = choco.exe list --source $ProdRepo -r | ConvertTo-ChocoObject

```

Comment:
--limit-output, otherwise 👍

Reply:
Best practices is `--source="'$TestRepo'"`

https://chocolatey.org/docs/commands-reference#scripting-integration---best-practices-style-guide

Reply:
If exclusively PowerShell, you can use apostrophes in the scripts as long as it is not using variables (it might remove interpolation).

Reply:
Updated all of the choco command lines to follow best practice.